docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -18,85 +18,43 @@ We'll cover the following topics:
 
 ## Installation

```

Comment:
With fresh eyes, I feel like this should be "package installation". This whole guide to me is "openziti controller installation" and separating the concept feels useful